docs/docs/distributors/introduction/overview.mdx

```diff
@@ -8,80 +8,83 @@ description: Audius Protocol Documentation
 
 import useBaseUrl from '@docusaurus/useBaseUrl'
 
-:::tip Looking for how to accept deliveries from your label or distributor?
-
-Come back soon and check the [Audius Support Page](https://support.audius.co) for more information.
-
-:::
-
 Audius supports bulk content ingestion via the DDEX standard.
 
 Providers, labels & partners with their own delivery infrastructure should reference the following
 documentation for guidance on how to send and administer content on Audius.
 
+:::tip Build it yourself
+
 If you're looking to build your own ingestion tooling, get started with the [Audius SDK](/sdk).
 
+:::
+
 ## What is DDEX?
 
-Digital Data Exchange (or "DDEX") is an international standards-setting organization that was formed
-in 2006 to develop standards that enable companies to communicate information along the digital
-supply chain more efficiently by:
+[Digital Data Exchange](https://ddex.net/),"DDEX", is an international standards-setting
+organization that was formed in 2006 to develop standards that enable companies to communicate
+information along the digital supply chain more efficiently
+
+DDEX accomplishes this by:
 
 - Developing standard message and file formats (XML or flat-file)
 - Developing choreographies for specific business transactions
 - Developing communication protocols (SFTP or based on web services)
 - Working with industry bodies to create a more efficient supply chain
 
-:::info More Information
+<!-- prettier-ignore -->
+<figure className="screensnippet-wrapper">
+  <img src={useBaseUrl('/img/distributors/ddex-map.png')} />
+  <figcaption>DDEX "Tube Map"</figcaption>
+</figure>
 
-- Learn more on the official DDEX Website here: [https://ddex.net/](https://ddex.net/)
-- Looking for a deeper technical dive? Checkout the DDEX knowledge base here:
-  [https://kb.ddex.net/](https://kb.ddex.net/)
+---
 
-:::
+## How DDEX Works with Audius
+
+The DDEX standard is a common language spec, in XML format, used to share releases, streaming, and
+royalty data.
+
+the same stack we’re using to work with traditional distributors to get music on-chain.
+
+Record Labels and other rights holders can work with their existing distribution pipelines pretty
+seamlessly and get their tunes up on audius.
+
+## Common Flow
```

Comment:
I think we should nix this section - it's more about the network take than the distributor. The distributor actually has the ability to control who gets paid out for the tracks they distribute - sometime this might be the artist/label/else